File: src/test/scala/cern/colt/matrix/tdouble/impl/DenseDoubleMatrix2DViewTest.scala
package cern.colt.matrix.tdouble.impl

import cern.colt.matrix.MatrixTypes.DenseDoubleMatrix2D

class DenseDoubleMatrix2DViewTest(arg0: String) extends DenseMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new DenseDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new DenseDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new DenseDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}

File: src/test/java/cernx/colt/matrix/tdouble/impl/DenseDoubleMatrix2DViewTest.java
package cernx.colt.matrix.tdouble.impl;

public class DenseDoubleMatrix2DViewTest extends DenseDoubleMatrix2DTest {

    public DenseDoubleMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
